storage/txn.go

```diff
@@ -193,7 +193,12 @@ func (t *txn) Update(ctx context.Context, key string, value []byte, origRev uint
 	panic("not implemented") // TODO: Implement
 }
 
-var _ Txn = &txn{}
```

Comment:
IMO no need for change unless you want to separate the transaction code into a separate package from the storage package. The original line is the typical way just for checking if the `txn` type has implemented `Txn` interface during compiling time. Let's keep it what it was.

Reply:
sure, reverted